### ISSUE TITLE
{June} Research & Implement Runic VM SDK into Browser

### ISSUE THREAD

**Issue by kthakore** [hotg-ai/rune#94]
Make rune work on Browser with all 4 runes.

- [x] Linking everything up with webassembly runtime
- [x] Executing tf.js
- [x] Capabilities from JS / Browser
- [ ] update tfjs-tflite integration into our RuneVM js
- [ ] Part of tutorial: Put this into a PWA app

**Comment by kthakore** [comment]
```
// index.tsx
import * as React from 'react'
import {render} from "react-dom"
// Adds the CPU backend.
import '@tensorflow/tfjs-backend-cpu';
// Import @tensorflow/tfjs-core
import * as tf from '@tensorflow/tfjs-core';
// Import @tensorflow/tfjs-tflite.
import {loadTFLiteModel, TFLiteModel} from '@tensorflow/tfjs-tflite';

import binaryDataUrl from './sin_model.tflite';

fetch(binaryDataUrl)
  .then(r => r.arrayBuffer())
  .then(array => {
    const blob = new Blob([array]);
    const objectURL = URL.createObjectURL(blob);
    loadTFLiteModel(objectURL);
  });

render(<h1>Hi there</h1>, document.getElementById("root"))``` 

Make this

**Comment by Michael-F-Bryan** [comment]
This was resolved in #249.
